src/datasources/__tests__/collectionDraft.test.js

```diff
@@ -61,7 +63,7 @@ describe('collectionDraft', () => {
       }
     }, {
       headers: {
-        'Client-Id': 'eed-test-graphql',
+        'Client-Id': 'mmt-react-ui',
```

Comment:
I think the client-id shouldn't change.   The client isn't a react ui, it is  a eed test graph client.